sklearn/metrics/scorer.py

```diff
@@ -290,7 +307,7 @@ def make_scorer(score_func, greater_is_better=True, needs_proba=False,
     Examples
     --------
     >>> from sklearn.metrics import fbeta_score, make_scorer
-    >>> ftwo_scorer = make_scorer(fbeta_score, beta=2)
+    >>> ftwo_scorer = make_scorer(fbeta_score, name='fbeta_score', beta=2)
```

Comment:
Aside: the `name` parameter to `make_scorer` seems clunky to me. Is there any interest in the future to make sure that all scikit-learn scorers have a `__name__` attribute?